.github/workflows/new-pr.yaml

```diff
@@ -0,0 +1,29 @@
+---
+name: New PR
```

Comment:
Why is this workflow needed?

Reply:
- A new workflow has been created to convert a new open PR into Draft mode
- Workflows(Build, Test, Lint, Docs) do not start if the PR is in draft mode

The idea is not starting all pipelines since the beginning and only when the developer considers that it is mature enough.